app/controllers/providers/proceeding_merits_task/does_client_have_parental_responsibilities_controller.rb

```diff
@@ -7,7 +7,10 @@ def show
 
       def update
         @form = Providers::ProceedingMeritsTask::ParentalResponsibilitiesForm.new(form_params)
-        return redirect_to providers_merits_task_list_is_client_child_subject_path(merits_task_list_id) if @form.relationship_to_child.eql?("false")
+        if @form.relationship_to_child.eql?("false")
```

Comment:
Is there a reason why this controller doesn't have the && !draft_selected and the other 2 do? Do we need to add it?

Reply:
Added